apps/dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1,6 +1,6 @@
 // Load bootstrap/font-awesome sprockets first
 @import "custom";
-@import "bootstrap";
+@import "../node_modules/bootstrap/scss/bootstrap";
```

Comment:
Just do `@import "bootstrap/scss/bootstrap";`. The `node_modules` directory is part of the sprockets load path.